packages/twenty-server/src/workspace/workspace-query-runner/utils/parse-result.util.ts

```diff
@@ -39,6 +39,8 @@ export const parseResult = (obj: any): any => {
     if (obj.hasOwnProperty(key)) {
       if (typeof obj[key] === 'object' && obj[key] !== null) {
         result[key] = parseResult(obj[key]);
+      } else if (key === '__typename') {
```

Comment:
@charlesBochet Hmm didn't try to get the `__typename`, what's the goal of that ?

Reply:
In case of a custom table, the table name is alias with an _ prefix. Pg_graphql returns typename with a _prefix too. These are returned in the response